app/views/mailers_common/_booking_confirmation.html.haml

```diff
@@ -114,3 +114,6 @@
 
 %p
   %small Visit ID: #{@visit.visit_id}
+
+%script{type: 'application/ld+json'}
```

Comment:
I'm concerned about this. This puts a `script` tag in the email. I thought webmail clients strip these from email as a security measure. Does this work?